SPEC.md

```diff
@@ -117,34 +120,62 @@ The dependencies are applied in order and each image dependency can overwrite fi
 Execution details specified in image dependencies are ignored.
 An optional *path whitelist* can be provided, in which case all non-specified files from all dependencies will be omitted in the final, assembled rootfs.
 
-Image Format TODO
+## App Container Executor
 
-* Define the garbage collection lifecycle of the container filesystem including:
+The **App Container Executor** defines the process by which applications contained in ACIs are executed.
+There are two "perspectives" in this process.
+The "*executor*" perspective consists of the steps that the App Container Executor (ACE) must take to set up the environment for the pod and applications.
+The "*app*" perspective is how the app processes inside the pod see the environment.
+
+## Pods
+
+The deployable, executable unit in the App Container specification is the **pod**.
+A **pod** is a list of apps that should be launched together inside a shared context.
+The shared context is defined as the conjunction of the following Linux namespaces (or equivalents on other operating systems):
+- PID namespace
+- network namespace
+- mount namespace
+- IPC namespace
+- UTS namespace
+The context may also consist of one or more isolators.
+
+Each app in a pod will start chrooted into its own unique read-write rootfs before execution.
+
+The definition of the **pod** - namely, the list of constituent apps, and any isolators that should apply to the entire pod - is codified in a [Pod Manifest](#pod-manifest-schema).
+Pod Manifests can serve the role of both _deployable template_ and _runtime manifest_: a template can be a candidate for a series of transformations before execution.
+For example, a Pod Manifest might reference an app with a label requirement of `version=latest`, which another tool might subsequently resolve to a specific version.
+Another example would be that volumes are "late-bound" by the executor; alternatively, an executor might add annotations.
+
+A Pod Manifest must be fully resolved (_reified_) before execution.
+Specifically, a Pod Manifest must have all `mountPoint`s satisfied by `volume`s, and must reference all applications deterministically (by image ID).
+At runtime, the reified Pod manifest is exposed to applications through the metadata service.
+
+Pod TODO:
```

Comment:
If start and stop order of apps within a pod is an executor implementation detail, that should probably be briefly clarified.